setup.py

```diff
@@ -48,6 +48,7 @@
                 'matplotlib',
                 'scikit-learn <= 1.1.3',  # Pinned due to boken scikit-optimize
                 'scikit-optimize',
+                'tqdm'
```

Comment:
Add comma at the end of list items as good practice. It won't show that this line was changed(comma added) after someone add new item.

Reply:
I see, thanks.